napari/_app_model/actions/_layerlist_context_actions.py

```diff
@@ -130,6 +130,17 @@
         ),
         menus=[LAYERCTX_SPLITMERGE],
     ),
+    Action(
```

Comment:
Could we move this above `Merge to Stack`? That way it sits just below `Split RGB` which I think would be nice.